source/scripts/accessibility.js

```diff
@@ -105,6 +105,9 @@ function keyControls (e) {
     case keys.N:
       document.getElementById(RESET_NO_ID).click();
       break;
+    case keys.Q:
+      document.getElementById('info-open-button').click();
+      break;

```

Comment:
great work adding keyboard shortcut for tutorial page, we need to reflect this in the readme